apps/src/templates/sectionProgressV2/LessonDataCell.jsx

```diff
@@ -7,13 +7,20 @@ import {lessonHasLevels} from '../progress/progressHelpers';
 import {ITEM_TYPE} from './ItemType';
 import ProgressIcon from './ProgressIcon';
 
-export default function LessonDataCell({lesson, studentLessonProgress}) {
+export default function LessonDataCell({
+  lesson,
+  studentLessonProgress,
+  addExpandedLesson,
+}) {
   const noLevels = !lessonHasLevels(lesson);
   const finished = studentLessonProgress?.completedPercent === 100;
   const partiallyComplete = studentLessonProgress && !finished;
 
   return (
-    <div className={classNames(styles.gridBox, styles.gridBoxLesson)}>
+    <div
+      className={classNames(styles.gridBox, styles.gridBoxLesson)}
+      onClick={() => addExpandedLesson(lesson.id)}
```

Comment:
I notice that the header only has a click action when the lesson has levels. Seems wise to do the same here, no?